packages/dds/tree/src/test/memory/.mocharc.cjs

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file for memory profiling tests
+ */
+
+const baseConfig = require("../../../.mocharc.cjs");
+
+const baseNodeOptions =
+	baseConfig["node-option"] !== undefined
+		? Array.isArray(baseConfig["node-option"])
+			? baseConfig["node-option"]
+			: [baseConfig["node-option"]] // If string, wrap as array to use spread operator
+		: []; // If undefined, use an empty array
+
+const extendedConfig = {
+	...baseConfig,
+	"fgrep": ["@Benchmark", "@MemoryUsage"],
+	"node-option": [...baseNodeOptions, "expose-gc", "gc-global", "unhandled-rejections=strict"], // without leading "--"
+	"reporter": "@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js", // Changed reporter option to use the memory test reporter.
+	"reporterOptions": ["reportDir=.memoryTestsOutput/"],
+	"timeout": "300000",
```

Comment:
https://dev.azure.com/fluidframework/public/_build/results?buildId=277969&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=5cfe65a4-d7cb-5934-e953-1c67617821f4&l=25243

@alexvy86 

Test failing due to `Timeout of 2000ms exceeded` which doesn't match the `300000` I've set in this config. Could this be overridden somewhere else?

https://github.com/microsoft/FluidFramework/blob/73b7b99e7b296055f445cc397f87df5283fcef0a/packages/test/mocha-test-setup/mocharc-common.cjs#L61


Maybe I explictly include `--timeout FOO` inside the `test:memory` script?

Reply:
As per our discussion, adding `timeout()` at the end of the test case solved the issue. CI on mocha test is now passing.